src/components/Icon/Icon.tsx

```diff
@@ -238,6 +238,11 @@ function Icon({
     contentMarkup = <div className={styles.Placeholder} />;
   } else if (isBundledIcon(source)) {
     SourceComponent = BUNDLED_ICONS[source];
+    const componentImportName = importForStringIcon(source);
+    // eslint-disable-next-line no-console
+    console.warn(
+      `Deprecation: passing the string "${source}" is deprecated and will be removed in the next major version. Pass in the "${componentImportName}" React Component from the @shopify/polaris-icons package instead.`,
```

Comment:
For the sake of consistency, I think we should continue translating these until we make the blanket decision not to

Reply:
> until we make the blanket decision not to

I thought we made the decision to not translate these? I think it is largely useless as our Polaris docs are not translated.

Reply:
10-4, I think I misinterpreted that as saying we should keep translating these, and work to translate our docs.